### PR TITLE
Added Fosscord-server 

### PR DESCRIPTION
**Pull request by omxpro**
### All Submissions:

* [ yes] Have you followed the guidelines in our Contributing document?
* [ yes] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [yes ] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### New Server Submissions:

1. [ yes] Does your submission pass tests (server is connectable)?
2. [ no] Does your server use a custom docker image?
    * [ no cuz it allready works] Did you PR the necessary changes to make it work?
3. [ yes] Have you added the server to the main README.md?
4. [ yes] Have you added a unique README.md for the server you are adding?

### CODE REVIEW

File: software/fosscord-server/egg-fosscord.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-11-26T10:45:23-05:00",
+    "name": "Fosscord",
+    "author": "chris@chrischromeyt.com",
+    "description": "https:\/\/github.com\/fosscord\/fosscord-server",
+    "features": null,
+    "images": [
+        "ghcr.io\/parkervcp\/yolks:nodejs_17"
```
Comment (Software-Noob) [inline software/fosscord-server/egg-fosscord.json:13]: Would rather stick to LTS releases, meaning 14 or 16. The minimum requirement for fosscord is node 14.

File: software/fosscord-server/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Fosscord-Server
+
+### From the [Fosscord-server](https://github.com/fosscord/fosscord-server) GitHub
+
+Run [Selfhosted discord](dev.fosscord.com), its selfhosted version of discord thats not connected to discord.com it has api,cdn and all that good stuff, for more information, checkout docs
```
Comment (Software-Noob) [inline software/fosscord-server/README.md:5]: Using the official description of the project would be more fitting than "all that good stuff"

File: README.md [hunk 3]
```diff
@@ -247,6 +247,8 @@ If you are reading this it looks like you are looking to add an egg to your serv
 ## [Software](/software/)
 ### Code Server
 * [Code-Server](/software/code-server)
+### Fosscord-server
+* [Fosscord-server](/software/fosscord-server)
```
Comment (Software-Noob) [inline README.md:251]: Please follow the alphabetical listing, which would put this after Elasticsearch

File: software/fosscord-server/egg-fosscord.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n# NodeJS Bot Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\napt update\r\napt install -y git curl jq file unzip make gcc g++ python python-dev libtool\r\n\r\nmkdir -p \/mnt\/server\r\ncd \/mnt\/server\r\n\r\ngit clone https:\/\/github.com\/fosscord\/fosscord-server .\r\n\r\ncd bundle\r\necho \"Building Fosscord\"\r\necho \"PORT=$SERVER_PORT\" > \/mnt\/server\/bundle\/.env\r\nmkdir \/mnt\/server\/api\/assets\/plugins\r\n\/usr\/local\/bin\/npm run setup\r\n\r\necho -e \"install complete\"\r\nexit 0",
+            "container": "node:14-buster-slim",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "Threads",
```
[the remark this answers is not in the document]
Comment (ChrisChrome) [reply software/fosscord-server/egg-fosscord.json:32]: This variable IS used by the fosscord-server itself
It reads the environment variable directly, not through a file or anything

[the remark this answers is not in the document]
Comment (ChrisChrome) [reply software/fosscord-server/egg-fosscord.json:32]: To clarify, it specifies the number of CPU threads to use, though most of the time 1 thread is fine

File: software/fosscord-server/README.md (hunk 2 above)
Comment (omxpro) [reply software/fosscord-server/README.md:5]: > Using the official description of the project would be more fitting than "all that good stuff"

I'll do that rn

File: README.md (hunk 3 above)
Comment (omxpro) [reply README.md:251]: > Please follow the alphabetical listing, which would put this after Elasticsearch

Kk

**Comment by ChrisChrome** [comment]
Listen, I'm the original author of this egg, it was never meant to be published here, at least not in it's current state
Can a maintainer or something please close this PR?
Once more work has been done on it I myself will open a new PR for it at a later date

**Comment by omxpro** [comment]
> Listen, I'm the original author of this egg, it was never meant to be published here, at least not in it's current state Can a maintainer or something please close this PR? Once more work has been done on it I myself will open a new PR for it at a later date

Done :) I am so sorry btw

**Pull request closed without merging**